api/libproject/devstart.rb

```diff
@@ -1611,6 +1619,9 @@ def populate_ops_user_affiliations(cmd_name, *args)
   gradle_args = ([
       ["--import-filename", op.opts.importFilename]
   ]).map { |kv| "#{kv[0]}=#{kv[1]}" }
+  gradle_args += ([
```

Comment:
I'm assuming there's a more-idiomatic-Ruby way to do this - happy to replace it with any suggestions.

Reply:
looks like that worked, thanks!